ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -27,8 +27,10 @@
 #include <unistd.h>
 #include <poll.h>
 
+#include <sys/mman.h>
 #include <sys/eventfd.h>
```

Comment:
When I compile these patches in our environment, I am getting new compiler warnings:
warning: implicit declaration of function 'unlink' [-Wimplicit-function-declaration]
warning: implicit declaration of function 'posix_fallocate' [-Wimplicit-function-declaration]
warning: implicit declaration of function 'close' [-Wimplicit-function-declaration]

I think you missed to add some header files